build.sbt

```diff
@@ -0,0 +1,64 @@
+lazy val scala212 = "2.12.10"
+lazy val scala213 = "2.13.0"
+
+lazy val `zio-ftp` = project
```

Comment:
I propose to restructure `build.sbt` and rely to helpers like [this](https://github.com/zio/zio-keeper/blob/master/project/BuildHelper.scala). It would be aligned with other libs, and simplify its maintenance.